civihr_default_theme/assets/sass/partials/_navs.scss

```diff
@@ -0,0 +1,23 @@
+// Styles for Navs
+// --------------------------------------------------
+
+.chr_tabs__header {
+    .chr_tabs__header-item {
+        .btn {
```

Comment:
Disregard, for now, this comment. Please see my comment about this component on https://github.com/compucorp/civihr-employee-portal/pull/117